Tools/install.sh

```diff
@@ -243,7 +252,9 @@ if [ -f "/usr/bin/clang-format-8" ]; then
     sudo update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-8 100
 fi
 
-python3 -m pip install --upgrade pip protobuf -r "$SMING_HOME/../Tools/requirements.txt"
+PYTHON_INSTALL_CMD="python3 -m pip install --upgrade pip protobuf -r \"$SMING_HOME/../Tools/requirements.txt\""
```

Comment:
Should `protobuf` be added to `requirements.txt` ?

Reply:
Yep, seems reasonable to me. I will commit the change shortly.